src/families/cosmos/utils.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import type {
+  CosmosDelegation,
+  CosmosDelegationInfo,
+  CosmosValidatorItem,
+  CosmosFormattedDelegation,
+} from "./types";
+import type { Unit } from "../../types";
+
+export function formatDelegations(
+  delegations: CosmosDelegation[],
+  validators: CosmosValidatorItem[]
+): CosmosFormattedDelegation[] {
+  return delegations.map((d) => ({
+    validator: validators.find(
+      (v) => v.validatorAddress === d.validatorAddress
+    ),
+    address: d.validatorAddress,
+    amount: d.amount,
+    pendingRewards: d.pendingRewards,
+    status: d.status,
+  }));
+}
+
+export const formatDelegationsInfo = (
+  delegations: CosmosDelegationInfo[],
+  validators: CosmosValidatorItem[]
+): {
+  validator: ?CosmosValidatorItem,
+  address: string,
+  amount: BigNumber,
+}[] => {
```

Comment:
I don't understand what this is VS the other. we have a lot of types that are a bit hard to understand at first glance.